setup.py

```diff
@@ -13,27 +13,19 @@
 # limitations under the License.
 
 import os
-import sys
-from datetime import date
 from typing import Dict, List, Tuple
 
 from setuptools import find_packages, setup
 
 
+is_release = False
```

Comment:
Mark, can is_release here read from version.py so that we don't keep it in two places? i.e. we only need to modify it once in version.py for nightly or regular builds.

Reply:
this does read from version.py, I just put a default in here so that code editors don't complain about an unamed variable. Once the later exec version.py file runs it overrides the is_release variable

Reply:
@dhuangnm, I've added a bit more documentation in setup.py to make this apparent